core/import-csv.h

```diff
@@ -17,11 +17,19 @@ enum csv_format {
 
 #define MAXCOLDIGITS 10
 
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
interesting - we include this in divelogimportdialog.cpp
I wonder why this didn't cause us problems...

Reply:
Now I know why. We don't need that include and should remove it.
Which means that technically we don't need that `#ifdef` bracing, but it's not wrong, either.

Reply:
It worked because the extern "C" was taken from dive.h.